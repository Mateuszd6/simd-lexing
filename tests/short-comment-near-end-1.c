HELLO_FUNCTION int random_function_foo(foobar_t xx){
  int xx = RESULT_OK;
  xx = random_function_fobar(db, "random_strg", &stmtModule, 0);
  return xx;
}

__declspec(dllexport)
HELLO__API int random_function_b(
  foobar_t xx,
  foo **foo_bars,
  const random_random_stuffs *scoo
){
  int xx = RESULT_OK;
  random_function_foobar(scoo);
  xx = random_function_foo(db);
  return xx;
}

/*****************************************************************************/
/* what am i even doing here itpointless */
// HELLO__API const char *random_function_(void){ return HELLO__API const char *random_function_(void){ return HELLO__API const char *random_function_(void){
return_HELLO__API
const char *random_function_(void){ return_HELLO__API const char // trolololololololo(void){ return qoiej qweoijqweo jqw eoiqj weoiqj eoiqjw         RANDOM_EXTENSION; }
 hello char
