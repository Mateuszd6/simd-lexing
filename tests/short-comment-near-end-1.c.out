../tests/short-comment-near-end-1.c:1:1: TOK 0 (L = 14): "HELLO_FUNCTION"
../tests/short-comment-near-end-1.c:1:16: TOK 0 (L = 3): "int"
../tests/short-comment-near-end-1.c:1:20: TOK 0 (L = 19): "random_function_foo"
../tests/short-comment-near-end-1.c:1:39: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:1:40: TOK 0 (L = 8): "foobar_t"
../tests/short-comment-near-end-1.c:1:49: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:1:51: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:1:52: TOK 1 (L = 1): "{"
../tests/short-comment-near-end-1.c:2:3: TOK 0 (L = 3): "int"
../tests/short-comment-near-end-1.c:2:7: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:2:10: TOK 1 (L = 1): "="
../tests/short-comment-near-end-1.c:2:12: TOK 0 (L = 9): "RESULT_OK"
../tests/short-comment-near-end-1.c:2:21: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:3:3: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:3:6: TOK 1 (L = 1): "="
../tests/short-comment-near-end-1.c:3:8: TOK 0 (L = 21): "random_function_fobar"
../tests/short-comment-near-end-1.c:3:29: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:3:30: TOK 0 (L = 2): "db"
../tests/short-comment-near-end-1.c:3:32: TOK 1 (L = 1): ","
../tests/short-comment-near-end-1.c:3:35: TOK 5 (L = 11): "random_strg"
../tests/short-comment-near-end-1.c:3:47: TOK 1 (L = 1): ","
../tests/short-comment-near-end-1.c:3:49: TOK 1 (L = 1): "&"
../tests/short-comment-near-end-1.c:3:50: TOK 0 (L = 10): "stmtModule"
../tests/short-comment-near-end-1.c:3:60: TOK 1 (L = 1): ","
../tests/short-comment-near-end-1.c:3:62: TOK 2 (L = 1): "0"
../tests/short-comment-near-end-1.c:3:63: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:3:64: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:4:3: TOK 0 (L = 6): "return"
../tests/short-comment-near-end-1.c:4:10: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:4:12: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:5:1: TOK 1 (L = 1): "}"
../tests/short-comment-near-end-1.c:7:1: TOK 0 (L = 10): "__declspec"
../tests/short-comment-near-end-1.c:7:11: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:7:12: TOK 0 (L = 9): "dllexport"
../tests/short-comment-near-end-1.c:7:21: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:8:1: TOK 0 (L = 10): "HELLO__API"
../tests/short-comment-near-end-1.c:8:12: TOK 0 (L = 3): "int"
../tests/short-comment-near-end-1.c:8:16: TOK 0 (L = 17): "random_function_b"
../tests/short-comment-near-end-1.c:8:33: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:9:3: TOK 0 (L = 8): "foobar_t"
../tests/short-comment-near-end-1.c:9:12: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:9:14: TOK 1 (L = 1): ","
../tests/short-comment-near-end-1.c:10:3: TOK 0 (L = 3): "foo"
../tests/short-comment-near-end-1.c:10:7: TOK 1 (L = 1): "*"
../tests/short-comment-near-end-1.c:10:8: TOK 1 (L = 1): "*"
../tests/short-comment-near-end-1.c:10:9: TOK 0 (L = 8): "foo_bars"
../tests/short-comment-near-end-1.c:10:17: TOK 1 (L = 1): ","
../tests/short-comment-near-end-1.c:11:3: TOK 0 (L = 5): "const"
../tests/short-comment-near-end-1.c:11:9: TOK 0 (L = 20): "random_random_stuffs"
../tests/short-comment-near-end-1.c:11:30: TOK 1 (L = 1): "*"
../tests/short-comment-near-end-1.c:11:31: TOK 0 (L = 4): "scoo"
../tests/short-comment-near-end-1.c:12:1: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:12:2: TOK 1 (L = 1): "{"
../tests/short-comment-near-end-1.c:13:3: TOK 0 (L = 3): "int"
../tests/short-comment-near-end-1.c:13:7: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:13:10: TOK 1 (L = 1): "="
../tests/short-comment-near-end-1.c:13:12: TOK 0 (L = 9): "RESULT_OK"
../tests/short-comment-near-end-1.c:13:21: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:14:3: TOK 0 (L = 22): "random_function_foobar"
../tests/short-comment-near-end-1.c:14:25: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:14:26: TOK 0 (L = 4): "scoo"
../tests/short-comment-near-end-1.c:14:30: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:14:31: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:15:3: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:15:6: TOK 1 (L = 1): "="
../tests/short-comment-near-end-1.c:15:8: TOK 0 (L = 19): "random_function_foo"
../tests/short-comment-near-end-1.c:15:27: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:15:28: TOK 0 (L = 2): "db"
../tests/short-comment-near-end-1.c:15:30: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:15:31: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:16:3: TOK 0 (L = 6): "return"
../tests/short-comment-near-end-1.c:16:10: TOK 0 (L = 2): "xx"
../tests/short-comment-near-end-1.c:16:12: TOK 1 (L = 1): ";"
../tests/short-comment-near-end-1.c:17:1: TOK 1 (L = 1): "}"
../tests/short-comment-near-end-1.c:22:1: TOK 0 (L = 17): "return_HELLO__API"
../tests/short-comment-near-end-1.c:23:1: TOK 0 (L = 5): "const"
../tests/short-comment-near-end-1.c:23:7: TOK 0 (L = 4): "char"
../tests/short-comment-near-end-1.c:23:12: TOK 1 (L = 1): "*"
../tests/short-comment-near-end-1.c:23:13: TOK 0 (L = 16): "random_function_"
../tests/short-comment-near-end-1.c:23:29: TOK 1 (L = 1): "("
../tests/short-comment-near-end-1.c:23:30: TOK 0 (L = 4): "void"
../tests/short-comment-near-end-1.c:23:34: TOK 1 (L = 1): ")"
../tests/short-comment-near-end-1.c:23:35: TOK 1 (L = 1): "{"
../tests/short-comment-near-end-1.c:23:37: TOK 0 (L = 17): "return_HELLO__API"
../tests/short-comment-near-end-1.c:23:55: TOK 0 (L = 5): "const"
../tests/short-comment-near-end-1.c:23:61: TOK 0 (L = 4): "char"
../tests/short-comment-near-end-1.c:24:2: TOK 0 (L = 5): "hello"
../tests/short-comment-near-end-1.c:24:8: TOK 0 (L = 4): "char"
Parsed: 25 lines, 46 ids, 1 strings, 0 chars, 1 ints, 0 hex,   0 floats
